src/styles/custom-properties.less

```
@@ -19,9 +26,37 @@
   --ironsworn-spacer-lg: 7px;
   --ironsworn-spacer-xl: 10px;
 
+  // Range input elements
+
+  --ironsworn-range-height: 20px;
+  // range slider thumb
```

Weird that styleliint doesn't hate this whitespace.

```suggestion
  // Range input elements
  --ironsworn-range-height: 20px;

  // range slider thumb
```

---

huh, weird. not sure where that could be coming from, but stylelint seems less fluent with LESS than it does SCSS, overall. so it might cease to be a problem soon.